### PR TITLE
Add  Themes

### PR DESCRIPTION
**Pull request by hendrasob**
Hi again friend. Today i added a theme version. Namely the blood and blood-dark themes.

blood

blood-dark

### CODE REVIEW

**Comment by DenverCoder1** [comment]
Thanks again for contributing!

If possible, could you update the screenshots to not have whitespace on the edges?

You can get clean static images from the demo site when running locally, but I can include the image links here to make it easier:

https://user-images.githubusercontent.com/20955511/112644211-35731c00-8e56-11eb-831f-563c01014d60.png

image

https://user-images.githubusercontent.com/20955511/112644234-3c9a2a00-8e56-11eb-936b-49dbb57dd1f4.png

image

File: docs/themes/README.md [hunk 1]
```diff
@@ -254,6 +254,16 @@ To enable a theme, append `&theme=` followed by the theme name to the end of the
 
 ![image](https://user-images.githubusercontent.com/20955511/112301991-357eea80-8ca3-11eb-9808-6dd6223c7647.png)
 
+`blood`
+
+![image](https://user-images.githubusercontent.com/73984717/112638604-036fb300-8e72-11eb-867c-4251cd83891a.png)
```
Comment (DenverCoder1) [inline docs/themes/README.md:259]:
```suggestion
![image](https://user-images.githubusercontent.com/20955511/112644211-35731c00-8e56-11eb-831f-563c01014d60.png)
```

File: docs/themes/README.md [hunk 2]
```diff
@@ -254,6 +254,16 @@ To enable a theme, append `&theme=` followed by the theme name to the end of the
...
+
+`blood-dark`
+
+![image](https://user-images.githubusercontent.com/73984717/112638122-79bfe580-8e71-11eb-83de-c4ba453afe4a.png)
```
Comment (DenverCoder1) [inline docs/themes/README.md:263]:
```suggestion
![image](https://user-images.githubusercontent.com/20955511/112644234-3c9a2a00-8e56-11eb-936b-49dbb57dd1f4.png)
```

**Review by DenverCoder1 — APPROVED** [review]

**Comment by hendrasob** [comment]
Okay, no problem. And i as sorry. By the way, I have fixed the last commit

**Pull request merged**